### PR TITLE
[Text Fragment article] Add info about link generation and analytics, remove OT info

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add info about programmatic link generation.
- Add info about  obtaining text fragment links for analytics. 
- Remove info about past origin trial.

### CODE REVIEW

File: src/site/content/en/blog/text-fragments/index.md [hunk 1]
```diff
@@ -387,6 +387,47 @@ The Text Fragments feature can be polyfilled to some extent. We provide a
 the [extension](https://github.com/GoogleChromeLabs/link-to-text-fragment), for browsers that do not
 provide built-in support for Text Fragments where the functionality is implemented in JavaScript.
 
+### Programmatic Text Fragment link generation
+
+The [polyfill](https://github.com/GoogleChromeLabs/text-fragments-polyfill) contains a file
+`fragment-generation-utils.js` that you can import and generate Text Fragment links with as
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:393]:
```suggestion
The [polyfill](https://github.com/GoogleChromeLabs/text-fragments-polyfill) contains a file
`fragment-generation-utils.js` that you can import and use to generate Text Fragment links. This is
```

Comment (jpmedley) [reply src/site/content/en/blog/text-fragments/index.md:393]: The phrase was grammatically correct, but it sounded awkward to me.

File: src/site/content/en/blog/text-fragments/index.md [hunk 2]
```diff
@@ -387,6 +387,47 @@ The Text Fragments feature can be polyfilled to some extent. We provide a
...
+    '';
+  url += `#:~:text=${prefix}${textStart}${textEnd}${suffix}`;
+  console.log(url);
+}
+```
+
+### Obtaining Text Fragments for analytics purposes
+
+Plenty of sites use the fragment for routing, which is why browsers strip out Text Fragments
+as to not break those pages. There is an
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:420]:
```suggestion
so as to not break those pages. There is an
```

File: src/site/content/en/blog/text-fragments/index.md [hunk 3]
```diff
@@ -387,6 +387,47 @@ The Text Fragments feature can be polyfilled to some extent. We provide a
...
+[acknowledged need](https://github.com/WICG/scroll-to-text-fragment/issues/128)
+to expose Text Fragments links to pages, for example, for analytics purposes,
+but the proposed solution is not implemented yet.
+As a workaround for now, you can use the one-liner script below to extract
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:424]:
```suggestion
As a workaround for now, you can use the code below to extract
```

File: src/site/content/en/blog/text-fragments/index.md [hunk 4]
```diff
@@ -430,14 +471,9 @@ contrived to begin with and since its exploitation requires _very_ specific prec
...
 
-For sites that still wish to opt-out, we have proposed a
-[Document Policy](https://github.com/w3c/webappsec-feature-policy/blob/master/document-policy-explainer.md)
-header value that they can send, so user agents will not process Text Fragment URLs. Since Document
-Policy is not yet shipped, we are running an
-[origin trial](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md) to
-apply this policy as an intermediate solution. The
-[ForceLoadAtTop](https://developers.chrome.com/origintrials/#/view_trial/3253850730775183361) origin
-trial is running from Chrome version 83 to 85.
+For sites that still wish to opt-out, we support a
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:474]:
```suggestion
For sites that wish to opt-out, Chrome supports a
```

File: src/site/content/en/blog/text-fragments/index.md [hunk 5]
```diff
@@ -430,14 +471,9 @@ contrived to begin with and since its exploitation requires _very_ specific prec
...
+[Document Policy](https://wicg.github.io/document-policy/)
+header value that they can send, so user agents will not process Text Fragment URLs.
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:476]:
```suggestion
header value that they can send so user agents will not process Text Fragment URLs.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by tomayac** [comment]
This should be good to go, but probably simply fell through the cracks.

File: src/site/content/en/blog/text-fragments/index.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ subhead: |
 authors:
   - thomassteiner
 date: 2020-06-17
-updated: 2021-02-02
+updated: 2021-05-03
```
Comment (tomayac) [inline src/site/content/en/blog/text-fragments/index.md:11]:
```suggestion
updated: 2021-05-10
```

**Pull request merged**